Igalia/phpreport
# Add Timer to Task Form

### Description

This PR adds the Timer to the Task Form.

### Testing

1. Click Start Timer
2. It should start counting the logged time and set the "From" select to the current time
3. Click Stop Timer
4. It should stop counting the logged time and set the "To" select to the current time
5. The selects should be clearable
6. The selects should be disabled while the timer is running
7. You can also manually edit the From/To time by selecting a specific time withing 15min intervals or just writing the time.

### Screenshots/Videos
[Screencast from 2023-10-04 16-03-00.webm](https://github.com/Igalia/phpreport/assets/20492786/2e909528-2ec7-4b7c-86c8-766a2ac5ba89)
